src/vs/platform/log/common/log.ts

```diff
@@ -745,6 +747,17 @@ export function LogLevelToString(logLevel: LogLevel): string {
 	}
 }
 
+export function LogLevelToLocalizedString(logLevel: LogLevel): ILocalizedString {
+	switch (logLevel) {
+		case LogLevel.Trace: return { original: 'trace', value: nls.localize('trace', "Trace") };
+		case LogLevel.Debug: return { original: 'debug', value: nls.localize('debug', "Debug") };
+		case LogLevel.Info: return { original: 'info', value: nls.localize('info', "Info") };
+		case LogLevel.Warning: return { original: 'warn', value: nls.localize('warn', "Warning") };
+		case LogLevel.Error: return { original: 'error', value: nls.localize('error', "Error") };
+		case LogLevel.Off: return { original: 'off', value: nls.localize('off', "Off") };
```

Comment:
Orignal value should be same as the value in English. Here original value is `trace` where as value is `Trace`

Reply:
Done, assuming I understood you correctly.